timesheet_automatic_reminder/models/common_method.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import time
+from datetime import datetime, timedelta, date
+import pytz
+
+# from openerp import models, api
+
+DATETIME = '%Y-%m-%d %H:%M:%S'
+DATE = '%Y-%m-%d'
+TIME = '%H:%M:%S'
+
+
+def is_weekend(selected_date):
```

Comment:
@dongshukai As the name of this function the python file I would rather call it `timeweekend`